packages/terra-section-header/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Section Header
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-section-header.svg)](https://www.npmjs.org/package/terra-section-header)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+Section Header presentational component that provides an onClick callback and animateable status icon.
```

Comment:
nit: Drop the `and animateable status icon.` part since we scoped that out.

Reply:
Done
aa42014004dd17cacbdf78f5176b2a74dcfbff6a